sdk/simulator-service/tests/simulation.spec.ts

```diff
@@ -0,0 +1,67 @@
+// import { ChainFamilyMap, ChainInfo } from '@summerfi/sdk-common/chains'
```

Comment:
The test is fully commented out. If not used please remove it, or add a TODO indicating that we are missing a test